src/CoreWCF.Primitives/src/CoreWCF/Channels/ChannelBindingMessageProperty.cs

```diff
@@ -39,6 +39,34 @@ public ChannelBinding ChannelBinding
             }
         }
 
+        public static bool TryGet(Message message, out ChannelBindingMessageProperty property)
+        {
```

Comment:
This class isn't public in NetFx but it is in CoreWCF as it's used by the transports. As these methods are not public on NetFx I added them to ChannelBindingMessagePropertyHelper which is a shared class compiled into every package and is internal.